src/app/app.module.ts

```diff
@@ -1,16 +1,22 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule } from '@angular/forms'; // <-- NgModel lives here
```

Comment:
Should remove comment
```suggestion
import { FormsModule } from '@angular/forms';
```